### PR TITLE
localCI: vendor ciao-down

### LINKED ISSUE #278: localCI: vendor ciao-down
use ciao-down to spawn VM and test pull requests

### PR DESCRIPTION
**Pull request by devimc**
in order to support the use of VMs to test pull request
localCI should use ciao-down

fixes #278

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

File: cmd/localCI/Gopkg.lock [hunk 1]
```diff
@@ -1,6 +1,13 @@
-memo = "6741108163c25829de2a1320721785eac9e6a0af6714a59b8e8ea522382ed194"
+memo = "5217388a073fbb9d27a4d6da44598eaa256a9468e261fe45610cf66f6f338fe0"
 
 [[projects]]
+  name = "github.com/01org/ciao"
```
Comment (grahamwhaley) [inline cmd/localCI/Gopkg.lock]: Just to ask, do you need the top level here, or could/should it just pull the ciao-down path https://github.com/01org/ciao/tree/master/testutil/ciao-down ?

**Comment by grahamwhaley** [comment]
niggles, otherwise
lgtm

File: cmd/localCI/Gopkg.lock (hunk 1 above)
Comment (devimc) [inline cmd/localCI/Gopkg.lock]: ciao-down depends of ciao (sad but true)

**Comment by devimc** [comment]
seems like vendoring ciao-down won't be needed if this pr is merged 
https://github.com/clearcontainers/tests/pull/247

**Pull request closed without merging**